src/components/MovieDetails/MovieProvider.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface MovieContext {
+    activeMovie: number | null;
```

Comment:
you just have `number | null` in multiple places. Maybe you can create separate type for this?

Reply:
This is fixed